recipe/build.sh

```diff
@@ -12,7 +12,7 @@ export GRPC_PYTHON_BUILD_WITH_CYTHON="True"
 if [[ `uname` == 'Darwin' ]]; then
     export CC=$(basename "${CC}")
     export PATH="$SRC_DIR:$PATH"
-    export GRPC_PYTHON_LDFLAGS=" -framework CoreFoundation"
+    export GRPC_PYTHON_LDFLAGS=" -framework CoreFoundation -read_only_relocs suppress"
```

Comment:
Trying this:

https://stackoverflow.com/questions/11317637/linker-error-when-unit-testing-ld-illegal-text-relocation-to-cstring-in-fr/20722340#20722340

Reply:
When I try this build locally, I get `ld: warning: -read_only_relocs cannot be used with x86_64`, so must be another trick needed.